server/sandbox_stop.go

```diff
@@ -16,7 +16,13 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	logrus.Debugf("StopPodSandboxRequest %+v", req)
 	sb, err := s.getPodSandboxFromRequest(req.PodSandboxId)
 	if err != nil {
-		return nil, err
+		if err == errSandboxIDEmpty {
```

Comment:
we aren't _always_ returning nil to kubelet for stop, we return err if the sandboxId is empty. I guess I haven't understood your comment :) could you clarify?